client/app/components/packages/landuse-form/show.hbs

```diff
@@ -1209,34 +1208,305 @@
+          <A.Prompt>
+            What is the location on site or in building?
+          </A.Prompt>
+          <A.Field>
+            {{landuseGeography.dcpLocationsiteinbuilding}}
+          </A.Field>
+        </Ui::Answer>
+
+
+      {{/each}}
       </section>
 
     {{/if}}
     {{/let}}
 
+    {{#let (intersect
+        (map-by "dcpActioncode" @package.landuseForm.landuseActions)
+        (array 'PP')
+      ) as |publicFacilitiesActions|}}
+      {{#if (gt publicFacilitiesActions.length 0)}}
+        <section class="form-section">
+          <h2 class="section-header">
+            <span id="disposition" class="section-anchor"></span>
+            Disposition
+          </h2>
 
-    <section class="form-section">
-      <h2 class="section-header">
-        <span id="disposition" class="section-anchor"></span>
-        Disposition [place holder]
-      </h2>
-    </section>
+          <Ui::Answer @beside={{true}} as |A|>
+            <A.Prompt>
+              What is the type of disposition?
+            </A.Prompt>
+            <A.Field>
+              {{optionset 'landuseForm' 'dcpTypedisposition' 'label' @package.landuseForm.dcpTypedisposition}}
+            </A.Field>
+          </Ui::Answer>
 
-    <section class="form-section">
-      <h2 class="section-header">
-        <span id="zoning-text-amendment" class="section-anchor"></span>
-        Zoning Text Amendment [place holder]
-      </h2>
-    </section>
+          {{#if (eq
+              @package.landuseForm.dcpTypedisposition
+              (optionset 'landuseForm' 'dcpTypedisposition' 'code' 'DIRECT')
+            )}}
+            <Ui::Answer @beside={{true}} as |A|>
+              <A.Prompt>
+                From which agency?
+              </A.Prompt>
+              <A.Field>
+                {{@package.landuseForm.dcpTextcityagency}}
+              </A.Field>
+            </Ui::Answer>
 
-    <section class="form-section">
-      <h2 class="section-header">
-        <span id="zoning-map-amendment" class="section-anchor"></span>
-        Zoning Map Amendment [place holder]
-      </h2>
-    </section>
+            <Ui::Answer @beside={{true}} as |A|>
+              <A.Prompt>
+                To whom?
+              </A.Prompt>
+              <A.Field>
+                {{@package.landuseForm.dcpTowhom}}
+              </A.Field>
+            </Ui::Answer>
+          {{/if}}
+
+        </section>
+      {{/if}}
+    {{/let}}
+
+    {{#let (intersect
+        (map-by "dcpActioncode" @package.landuseForm.landuseActions)
+        (array 'ZR')
+      ) as |publicFacilitiesActions|}}
+      {{#if (gt publicFacilitiesActions.length 0)}}
+        
+        <section class="form-section">
+          <h2 class="section-header">
+            <span id="zoning-text-amendment" class="section-anchor"></span>
+            Zoning Text Amendment
+          </h2>
+          {{#each @package.landuseForm.affectedZoningResolutions as |affectedZoningResolution|}}
```

Comment:
@dhochbaum-dcp , "List all affected Zoning Resolution Sections, indicating their title and number. This section only applies to ZR actions." is missing after line 1414...please check file zoning-text-amendment.hbs line 8. 

Could you please also check the other section to see if similar text is missing.